.github/workflows/ci.yml

```diff
@@ -29,16 +34,6 @@ jobs:
         env:
           BALLERINA_VERSION: ballerina-${{ matrix.ballerina-version }}-swan-lake
         run: |
-          # Download jq
-          mkdir -p ./bin
-          case ${{ matrix.os }} in
-            "ubuntu-latest") jq_bin="jq-linux64" ;;
-            "macOS-latest")  jq_bin="jq-osx-amd64" ;;
-          esac
-          curl -fsSL https://github.com/stedolan/jq/releases/download/jq-1.6/"$jq_bin" --output ./bin/jq
-          chmod u+x ./bin/jq
-          echo "$PWD/bin" >> $GITHUB_PATH
-
```

Comment:
jq is needed for the no-argument invocation of ./bin/verify-exercises -- this is the scenario when config.json is queried.

Reply:
It was a simple naming issue: https://github.com/exercism/ballerina/pull/179#issuecomment-1527463819